icon_validator/rules/plugin_validators/description_validator.py

```diff
@@ -35,7 +37,7 @@ def validate_dictionary(dict_, dict_key):
                 try:
                     DescriptionValidator.validate_description(value["description"])
                 except ValidationException as e:
-                    raise ValidationException(f"{dict_key} key '{key}'\'s description ends with a period when it should not.", e)
+                    raise ValidationException(f"{dict_key} key '{key}' error.", e)
```

Comment:
Keep the original description here. Generically saying error will drive me nuts trying to figure out what went wrong.